css/event-detail.css

```diff
@@ -0,0 +1,84 @@
+html {
```

Comment:
@TerryHCollins  Please delete this file for this branch. Thanks!

Reply:
The file has been removed!